.evergreen/config.yml

```diff
@@ -1702,6 +1702,32 @@ axes:
           GO_DIST: "/opt/golang/go1.15"
           PYTHON3_BINARY: python3
 
+  # OSes that require >= 4.0 for SSL
+  - id: os-ssl-40
```

Comment:
Only 3.6.20+ server configurations support Ubuntu 18.04 according to [production notes](https://docs.mongodb.com/v3.6/administration/production-notes/). We'll need a new Axis for 4.0 and up, so we can continue to use Ubuntu 16.04 on < 4.0 and Ubuntu 18.04 on >= 4.0. The versions of MacOS and Windows for this axis are the same as `os-ssl-32`.

Reply:
Discussed offline; 3.6 (even 3.6.23) cannot run on Ubuntu 18.04 because of some SSL issues (see setup failures [here](https://spruce.mongodb.com/version/6079bce52fbabe5a1cad93f3/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)), so we'll have to keep 3.6 and 4.0 separated unfortunately.